build.gradle.kts

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-FileCopyrightText: 2025 NewPipe e.V. <https://newpipe-ev.de>
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    alias(libs.plugins.google.protobuf) apply false
+}
+
+allprojects {
+    apply(plugin = "java-library")
+    apply(plugin = "maven-publish")
+
+    tasks.withType<JavaCompile> {
+        options.encoding = Charsets.UTF_8.toString()
+    }
+
+    extensions.configure<JavaPluginExtension> {
+        toolchain {
+            languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
Why Java 17 and not 11?

We used this before:
``` build.gradle
    sourceCompatibility = JavaVersion.VERSION_11
    targetCompatibility = JavaVersion.VERSION_11
```
The CI also works with Java 11.

Reply:
Fixed, thanks for noticing